src/test/interface/stansummary_test.cpp

```diff
@@ -411,7 +413,7 @@ TEST(CommandStansummary, check_csv_output) {
                                 + "interface" + path_separator
                                 + "example_output" + path_separator
                                 + "tmp_test_target_csv_file.csv";
-  std::string arg_csv_file = "--csv_file=" + target_csv_file;
```

Comment:
Where does the `csv_file` -> `csv_filename` change come from?

Reply:
it was a bug in the test - the option long name is `csv_filename` - maybe the boost library was ok with prefix?